tutorial/dashbio_component_examples.py

```diff
@@ -721,15 +930,18 @@
         },
         {
             'param_name': 'Point sizes and line widths',
-            'description': 'Change the sizeo f the points on the scatter plot, and the widths of the effect \
-            lines and the genome-wide line.',
+            'description': 'Change the sizeo f the points on the scatter plot, \
+            and the widths of the effect lines and the genome-wide line.',
```

Comment:
```suggestion
            and the widths of the effect lines and genome-wide line.',
```
if I understand correctly, i.e., "change the width of the effect lines and the width of the genome-wide line" (as is, the meaning is "change ... and change the genome-wide line" which doesn't seem fine, because we don't know which feature of the genome-wide line is getting changed).